examples/statistical_charts/src/main.rs

```diff
@@ -345,7 +345,7 @@ fn fully_styled_box_plot() {
         v
     };
 
-    let x_data = vec![
+    let x_data = [
```

Comment:
Do we just not need to declare `vec!` here?

Reply:
This comes from a `clippy` lint .